src/shims/ApiCompatBaseline.netcoreapp.netfx461.txt

```diff
@@ -28,6 +28,11 @@ MembersMustExist : Member 'Microsoft.SqlServer.Server.SqlMetaData..ctor(System.S
 MembersMustExist : Member 'Microsoft.SqlServer.Server.SqlMetaData..ctor(System.String, System.Data.SqlDbType, System.Type, System.String, System.Boolean, System.Boolean, System.Data.SqlClient.SortOrder, System.Int32)' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'Microsoft.SqlServer.Server.SqlMetaData.DbType.get()' does not exist in the implementation but it does exist in the contract.
 MembersMustExist : Member 'Microsoft.SqlServer.Server.SqlMetaData.Type.get()' does not exist in the implementation but it does exist in the contract.
+CannotMakeMemberNonVirtual : Member 'System.Data.Common.DBDataPermission.Add(System.String, System.String, System.Data.KeyRestrictionBehavior)' is non-virtual in the implementation but is virtual in the contract.
+CannotMakeMemberNonVirtual : Member 'System.Data.Common.DBDataPermission.CreateInstance()' is non-virtual in the implementation but is virtual in the contract.
```

Comment:
@danmosemsft Are you sure? I see virtual both in the implementation and in the ref. Maybe ApiCompat.txt is outdated